runtime/lua/vim/lsp.lua

```diff
@@ -584,8 +584,9 @@ do
   text_document_did_change_handler = function(_, bufnr, changedtick,
       firstline, lastline, new_lastline, old_byte_size, old_utf32_size,
       old_utf16_size)
-    local _ = log.debug() and log.debug("on_lines", bufnr, changedtick, firstline,
-    lastline, new_lastline, old_byte_size, old_utf32_size, old_utf16_size, nvim_buf_get_lines(bufnr, firstline, new_lastline, true))
+    local _ = log.debug() and log.debug("on_lines bufnr: "..bufnr..", changedtick: "..changedtick..", firstline: "..firstline..
```

Comment:
Probably better to do string.format I would guess, than so many concatenations.

Or, `vim.inspect({...})` or similar.